src/platform/packages/shared/response-ops/rule_params/custom_threshold/v1.ts

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import { schema } from '@kbn/config-schema';
+import { COMPARATORS } from '@kbn/alerting-comparators';
+import { dataViewSpecSchema } from '../common';
```

Comment:
I thought `dataViewSpecSchema` was going to be shared but if it is only used here maybe it does not make sense to be in `common`. If it is only used in one place it isn't "common" to anything.

Reply:
It'll be shared with metrics threshold rule params. But it'll be introduced by @benakansara in different PR.